### PR TITLE
[Feature]: Feature Flag Hide Camera Roll Keyboard

### PR DESCRIPTION
**Pull request by marcomaddy1982**
### Dependencies

JIRA: https://wearezeta.atlassian.net/browse/ZIOS-13660

### CODE REVIEW

**Comment by marcomaddy1982** [comment]
> Looks good to me 👍 How confident are you that we will not call any PHPhotoLibrary API? (also elsewhere in the app?)

I played a bit around with the app and everything seemed ok but if someone of you @johnxnguyen @typfel @billypchan @KaterinaWire @David-Henner know where the camera roll permission alert could be shown would be helpful.

**Comment by johnxnguyen** [comment]
You can access the camera roll via the sketch view, but I suppose that is another topic. Unless you consolidate all camera roll access flags into a single `cameraRoll` flag and use it to disable **all** access to the camera roll. For me this makes more sense, since I'm not sure why you would choose disable the camera roll on one screen and not on another (in the context of security).

**Comment by marcomaddy1982** [comment]
> You can access the camera roll via the sketch view, but I suppose that is another topic. Unless you consolidate all camera roll access flags into a single `cameraRoll` flag and use it to disable **all** access to the camera roll. For me this makes more sense, since I'm not sure why you would choose disable the camera roll on one screen and not on another (in the context of security).

Ok but should we do this in another PR or we directly on this PR?

**Comment by johnxnguyen** [comment]
> Ok but should we do this in another PR or we directly on this PR?

We can do it in another PR

File: Wire-iOS/Sources/Helpers/Bundle+SecurityFlags.swift [hunk 1]
```diff
@@ -52,6 +50,8 @@ enum SecurityFlags {
             return "CustomBackendEnabled"
         case .shareExtension:
             return "ShareExtensionEnabled"
+        case .cameraRoll:
+            return "CameraRollEnanbled"
```
Comment (typfel) [inline Wire-iOS/Sources/Helpers/Bundle+SecurityFlags.swift:54]:
```suggestion
            return "CameraRollEnabled"
```

**Pull request merged**